Pythonwin/pywin/idle/CREDITS.txt

```diff
@@ -0,0 +1,17 @@
+IDLEfork Credits
```

Comment:
From https://github.com/python/cpython/blob/44d3d1a394571c6ddfe29911f92fdd29fd62db71/Lib/idlelib/CREDITS.txt

Reply:
I think the stuff here might predate that, and this isn't a license, so we are making life more difficult by treating it as one - we need less of them 😅